TheBTeam.ConsoleApp/TransactionViewer.cs

```diff
@@ -35,34 +41,50 @@ public static void ViewTransactionAccordingCategory(List<Transaction> transactio
             var categoryOfTransaction = ConsoleFactory.GetCategoryOfTransaction();
             Console.WriteLine($"{categoryOfTransaction}");
             var tmpTransactions = transactions.Where(t => t.User.Email == email).Where(t => t.Category == categoryOfTransaction);
```

Comment:
This is logic, i thing it should be in Bll.TransactionService

Reply:
what more i Jira, task was: Wyszukiwanie transakcji po kategorii
no info about user